Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM rhel8/go-toolset:1.12.8 AS builder
+WORKDIR /opt/app-root/src/go/src/github.com/knative/client
+USER root
+COPY . .
+RUN TAG="v0.8.0" make build
+
+FROM ubi8:8-released
+COPY --from=builder /opt/app-root/src/go/src/github.com/knative/client/kn /usr/bin/kn
+
+LABEL \
+      com.redhat.component="openshift-serverless-1-tech-preview-client-kn-rhel8-container" \
+      name="openshift-serverless-1-tech-preview/client-kn-rhel8" \
+      version="0.8.0" \
+      summary="Red Hat OpenShift Serverless 1 - CLI client kn" \
+      description="CLI client 'kn' for managing Red Hat OpenShift Serverless 1" \
+      io.k8s.display-name="Red Hat OpenShift Serverless 1 - CLI client kn" \
+      io.k8s.description="CLI client 'kn' for managing Red Hat OpenShift Serverless 1" \
```

Comment:
I would like to see summary and display-name values replaced with something more consistent to what we use for Knative Serving. Perhaps "Red Hat OpenShift Serverless 1 Kn CLI"... or if you feel 'client' must be in the name, "Red Hat OpenShift Serverless 1 Kn Client CLI"?

Reply:
I could live with "Red Hat OpenShift Serverless 1 kn CLI" (but please with lowercase 'k' in 'kn' as this is how we refer to it in the documentation and how it is called on the command line).

Reply:
Changed to `Red Hat OpenShift Serverless 1 kn CLI`.